test/src/test/java/org/corfudb/runtime/BackupRestoreIT.java

```diff
@@ -271,7 +271,8 @@ private void compareCorfuStoreTables(CorfuStore corfuStore1, String tableName1,
         TarArchiveInputStream tarInput = new TarArchiveInputStream(fileInput);
         TarArchiveEntry entry;
         while ((entry = tarInput.getNextTarEntry()) != null) {
-            streamIDs.add(UUID.fromString(entry.getName()));
+            String streamId = entry.getName().substring(0, entry.getName().indexOf("."));
```

Comment:
`entry.getStreamId()`

Reply:
Note that entry is an instance of `TarArchiveEntry`

Reply:
sorry, didn't notice